unite_contranges.py

```diff
@@ -0,0 +1,147 @@
+#Allison Towner
+#09 October 2020
+
+#continuum_selections.py in analysis code is located at:
+#in ALMA_IMF/reduction/analysis/
+
+import numpy as np
+execfile('/orange/adamginsburg/atowner/ALMA_IMF/contranges/parse_contdotdat.py')
+
+#Read in metadata from .json file
+basepath = '/orange/adamginsburg/ALMA_IMF/2017.1.01355.L'
+with open(basepath + '/contdatfiles.json', 'r') as fh:
+    contdatfiles = json.load(fh)
+with open(basepath + '/metadata.json', 'r') as fh:
+    metadata = json.load(fh)
+
+
+#Set field and band name - this should be detected automatically in the final version of the script but is hardwired here for testing purposes
+field = 'G327.29'
+band = 'B3'
```

Comment:
to generalize this code, I think we just indent everything below and 
```python
def merge_contdotdats(field='G327.29', band='B3'):
```
and then run this on each combination of field/band

Reply:
I have made this change but left field='' and band=''. Also removed the hard-coded field and band names from the script